src/main/resources/db/changelog/changes/v1.0.0/sql/entity-type-definitions/insert-user-details-definition.sql

```diff
@@ -13,6 +13,89 @@ INSERT INTO entity_type_definition (id, derived_table_name, definition)
                       "labelAlias": "User active",
                       "visibleByDefault": false
                 },
+              {
+                       "name": "address_ids",
+                       "dataType": {
+                         "dataType":"arrayType",
+                         "itemDataType": {
+                           "dataType": "rangedUUIDType"
+                         }
+                       },
+                       "labelAlias": "Address Ids",
+                       "visibleByDefault": false
+                 },
+                 {
+                       "name": "address_type_names",
+                       "dataType":{
+                         "dataType":"arrayType",
+                         "itemDataType": {
+                           "dataType": "stringType"
+                         }
+                       },
+                       "labelAlias": "Address type names",
+                       "visibleByDefault": true,
+                       "idColumnName": "address_ids",
+                        "source": {
+                               "entityTypeId": "e627a89b-682b-41fe-b532-f4262020a451",
+                               "columnName": "addressType"
+                        }
+                 },
+                {
+                      "name": "cities",
+                      "dataType":{
+                        "dataType":"stringType"
```

Comment:
Shouldn't this be an array type? Same with region, country ID, postal code, etc

Reply:
@mweaver-ebsco, I totally forgot to update the changes over here. But addressed your comment now.